packages/clay-color-picker/src/__tests__/index.tsx

```diff
@@ -373,6 +396,86 @@ describe('Interactions', () => {
 			fireEvent.change(hexInput, {target: {value: 'DDDDDD'}});
 
 			expect(handleColorsChange).toBeCalledTimes(2);
+
+			expect(document.body).toMatchSnapshot();
+		});
+	});
+
+	describe('color editor changing color value', () => {
+		let editorGetByLabelText: any;
+		let editorGetByTitle: any;
+
+		afterEach(cleanup);
+
+		beforeEach(() => {
+			const {getByLabelText, getByTitle} = render(
+				<ClayColorPickerWithCustomColors />
+			);
+
+			editorGetByLabelText = getByLabelText;
+			editorGetByTitle = getByTitle;
+		});
+
+		it('does not update when a sploch is not clicked', () => {
+			const input = editorGetByLabelText(
+				/Color selection is/
+			) as HTMLInputElement;
+
+			fireEvent.change(input, {target: {value: 'DFCAFF'}});
+
+			const dropdownToggle = document.querySelector('.dropdown-toggle');
+
+			fireEvent.click(dropdownToggle as HTMLButtonElement, {});
+
+			const colorEditorToggle = (
+				document.body as HTMLElement
+			).querySelector('.clay-color-header button');
+
+			fireEvent.click(colorEditorToggle as HTMLButtonElement, {});
+
+			expect(document.body).toMatchSnapshot();
```

Comment:
Make sense! I will change using snapshots and try to make some asserts to the components that define the test case. It would be great to use those custom components that you have mentioned and the input that is on the bottom of the gradient and the hue one (`customHexInput`)